src/actions/change-methods/showPackagesQuickPick.ts

```diff
@@ -0,0 +1,18 @@
+import * as vscode from 'vscode';
+
+export default function showPackagesQuickPick({
+    projFileFullPath,
+    installedPackages,
+    packageRefSection,
+    parsed,
+    originalContents
+}: any): Thenable<any> {
+    return vscode.window.showQuickPick(installedPackages)
+        .then((selectedPackage: string | undefined) => ({
+            projFileFullPath,
+            selectedPackage,
+            parsed,
+            packageRefSection,
+            originalContents
+        }));
```

Comment:
You could make this more concise by using object destructuring/spreading. E.g., the function could receive `{ installedPackages, ...otherPackageData }` and the `then` callback could return `({ selectedPackage, ...otherPackageData })`. Maybe you're following old code style here (I can't recall whether I used destructuring/spreading much in other files)? In any case, I think the conciseness is better, maybe with an added type (instead of `any`) for the function argument.

Reply:
I'm very unfamiliar with destructuring, but I gave this a shot.  I'm open to further changes if needed!